src/NServiceBus.Testing/TestingContext.cs

```diff
@@ -12,31 +10,14 @@ public TestingContext(IMessageCreator messageCreator = null) : base(messageCreat
 
         public TestingContext(IMessageCreator messageCreator, TimeoutMessage<object>[] timeoutMessages) : base(messageCreator)
         {
-            this.timeoutMessages = new ConcurrentQueue<TimeoutMessage<object>>(timeoutMessages);
+            this.previousTimeouts = timeoutMessages;
         }
 
-        public TimeoutMessage<object>[] TimeoutMessages => timeoutMessages.ToArray();
```

Comment:
the code was basically moved down the hierarchy to be available out of the box for all testable context classes, not just this context which is designed for sagas (and to track timeouts across multiple contexts).